cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgreSQLSink.java

```diff
@@ -616,32 +616,32 @@ public String buildTableName(String servicePath, String entity, String entityTyp
                                 + "dm-by-service-path data model");
                     } // if
                     
-                    name = NGSIUtils.encode(servicePath, true, false);
+                    name = NGSIUtils.encodePostgresTable(servicePath);
```

Comment:
After the changes in this PR, is `encode()` still used in other parts of the code? Or it can be removed?

Reply:
encode() is used in below part  of the code:
When "enable encoding=false" that time encode() is used to build DBName and SchemaName in class NGSIPostgreSQLSink. Also it is used in other sink like NGSIPostgisSink, NGSIOracleSink,NGSIMysqlSink etc. So we can't remove it.

Reply:
Thanks for the clarification

NTC